deploy/csv2db.py

```diff
@@ -0,0 +1,23 @@
+import json
+import pandas as pd
+import subprocess
+from urllib.parse import urlparse
+
+from sqlalchemy import create_engine
+
+CSVPATH = 'tests/data.csv'
+
+command = 'heroku config:get DATABASE_URL'
+heroku_pg = subprocess.check_output(command.split()).decode('utf-8')
+url = urlparse(heroku_pg)
+config = {'host': url.hostname,
+          'database': url.path[1:-1],
+          'user': url.username,
+          'port': url.port, 
+          'password': url.password}
+
+engine = create_engine('postgres://', connect_args=config)
```

Comment:
Ah, right, of course.  You can still avoid parsing the URL, though: `create_engine(heroku_pg)` should work, no?